.github/workflows/test.yml

```diff
@@ -24,6 +24,15 @@ jobs:
       matrix:
         include:
           - container: ghcr.io/oca/oca-ci/py3.6-ocb12.0:latest
+            include: "macavrac_base"
+            makepot: "true"
+            name: test with OCB
+          - container: ghcr.io/oca/oca-ci/py3.6-ocb12.0:latest
+            include: "polln_shift"
+            makepot: "true"
+            name: test with OCB
+          - container: ghcr.io/oca/oca-ci/py3.6-ocb12.0:latest
+            exclude: "macavrac_base,polln_shift"
```

Comment:
Est-ce que ceci est auto-généré depuis copier ? Sinon, on peut juste retirer les tests de polln et macavrac

Reply:
yes, copier asks for rebel module groups. they are tested separately. i don’t know whether it is possible to simply skip some modules. i’d rather have modules tested separately instead of not at all, because that would mean that we wouldn’t be aware if they got broken by some commit.

Reply:
I think we can exclude them by removing the configuration from the test.yaml: there are no tests in those modules and we do not maintain them.

=> less time => less energy.